modules/terrain/docs/api-reference/terrain-loader.md

```diff
@@ -0,0 +1,73 @@
+# TerrainLoader
+
+The `TerrainLoader` parses the OBJ half of the classic Wavefront OBJ/MTL format.
+
+| Loader                | Characteristic                                |
+| --------------------- | --------------------------------------------- |
+| File Extension        | `.png`, `.pngraw`                             |
+| File Type             | Binary                                        |
+| File Format           | Encoded height map                            |
+| Data Format           | [Mesh](/docs/specifications/category-mesh.md) |
+| Decoder Type          | Asynchronous                                  |
+| Worker Thread Support | Yes                                           |
+| Streaming Support     | No                                            |
+
+## Usage
+
+```js
+import {ImageLoader} from '@loaders.gl/images';
+import {TerrainLoader} from '@loaders.gl/terrain';
+import {load, registerLoaders} from '@loaders.gl/core';
+
+registerLoaders(ImageLoader);
```

Comment:
Why are we registering the image loader? If the TerrainLoader depends on it, import it and use it.

Reply:
I put in the forced ImageLoader and it seems to work in the tests, so sgtm.

Reply:
`ImageLoader` cannot be bundled into a worker.